notebooks/api/0.8/11-container-images-k8s.ipynb

```diff
@@ -1037,10 +1036,6 @@
     "assert workerimage_opendp.built_at is not None, str(workerimage_opendp.__dict__)\n",
     "assert workerimage_opendp.image_hash is not None, str(workerimage_opendp.__dict__)\n",
     "\n",
-    "assert _images[workerimage_opendp.built_image_tag] == workerimage_opendp, str(\n",
```

Comment:
Q:  Is this assert statement not needed anymore?

Reply:
Refer to 
https://github.com/OpenMined/PySyft/pull/9298#discussion_r1780393223